lib/lightning/invocation.ex

```diff
@@ -81,14 +81,14 @@ defmodule Lightning.Invocation do
         {:type, type}, dynamic ->
           dynamic([d], ^dynamic and d.type == ^type)
 
-        {:date, date}, dynamic ->
-          dynamic([d], ^dynamic and fragment("date(?)", d.inserted_at) == ^date)
-
         {:datetime, ts}, dynamic ->
```

Comment:
Great! Thank you for removing this, this would require another index to be performant. The `lt` and `gt` combo is much better.